### PR TITLE
refactor: improve vaults and ironbank overview

### PR DESCRIPTION
**Pull request by MarcoWorms**
tackles parts of #150

### CODE REVIEW

**Review by KurogeWashu — APPROVED** [review]
lgtm!

File: docs/getting-started/products/yvaults/overview.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ## What are yVaults?
 
-[yVaults](https://yearn.finance/vaults) are like savings accounts for your crypto assets. They accept your deposit, then route it through a strategies which seek out the highest yield available in DeFi.
+[yVaults](https://yearn.finance/vaults) takes deposits then route it through strategies which seek out the highest yield available in DeFi. [yVaults and Strategies](https://medium.com/iearn/yearn-finance-explained-what-are-vaults-and-strategies-96970560432) compose our core product which is served like a "crypto savings account": Yearn users can deposit/withdraw funds in yVaults at any time they want, and while deposited funds will generate yield using the same crypto that was deposited. yVaults are the lowest-risk investing type our protocol offers, and it's a great choice for users that want to generate steady "uponly" yield using DeFi tokens.
```
Comment (saltyfacu) [inline docs/getting-started/products/yvaults/overview.md:7]: yVaults take~~s~~

deposits **and** then route ~~it~~ them

Please take another look to improve the redaction. IMO, it's all over the place

File: docs/getting-started/products/iron-bank/collateral-and-reserve-factor.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 ## Introduction
 
-The Iron Bank provides a lending service both to users and protocols. For protocols that are whitelisted this can be zero-collateral loans. For users, this will be an overcollateralised loan.
+The Iron Bank provides a lending service both to users and protocols. For protocols that are whitelisted this can be zero-collateral loans. For users, this will be an overcollateralised loan. Unlike [Yearn Vaults](https://docs.yearn.finance/getting-started/products/yvaults/overview), Iron Bank doesn't have automatic investment strategies and is a product that involves risk management from the user side in order to avoid losses or liquidations.
```
Comment (saltyfacu) [inline docs/getting-started/products/iron-bank/collateral-and-reserve-factor.md:7]: IB and yVaults are very different things. You can't compare both. IB is a lending market

**Review by saltyfacu — CHANGES_REQUESTED** [review]
please check the comments

**Comment by saltyfacu** [comment]
IB is a standalone product and should not be in our dos anymore. Closing this

**Pull request closed without merging**